config/schedule.rb

```diff
@@ -28,3 +28,12 @@
 every 1.day, at: '4:30 am' do
   rake 'batch:update_daily'
 end
+
+every 1.day, at: '12:30 am' do
+  rake 'cache:warm:homepage'
```

Comment:
This snuck back in. This no longer does anything useful, and I deleted that rake task.